### PR TITLE
chore(css): Remove reflexbox from accountSubscriptions

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/account/accountSubscriptions.jsx [hunk 1]
```diff
@@ -143,4 +130,22 @@ class AccountSubscriptions extends AsyncView {
...
+  width: 50%;
+  padding-right: ${space(2)};
+`;
+
+const SubscriptionName = styled('div')`
+  font-size: 1.2em;
+`;
+const Description = styled('div')`
+  font-size: 0.8em;
+  margin-top: 6px;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/account/accountSubscriptions.jsx]:
```suggestion
  margin-top: ${space(0.75)};
```

File: src/sentry/static/sentry/app/views/settings/account/accountSubscriptions.jsx [hunk 2]
```diff
@@ -143,4 +130,22 @@ class AccountSubscriptions extends AsyncView {
...
+const SubscriptionDetails = styled('div')`
+  width: 50%;
+  padding-right: ${space(2)};
+`;
+
+const SubscriptionName = styled('div')`
+  font-size: 1.2em;
+`;
+const Description = styled('div')`
+  font-size: 0.8em;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/account/accountSubscriptions.jsx]:
```suggestion
  font-size: ${p => p.theme.fontSizeSmall};
```

File: src/sentry/static/sentry/app/views/settings/account/accountSubscriptions.jsx [hunk 3]
```diff
@@ -143,4 +130,22 @@ class AccountSubscriptions extends AsyncView {
   }
 }
 
+const SubscriptionDetails = styled('div')`
+  width: 50%;
+  padding-right: ${space(2)};
+`;
+
+const SubscriptionName = styled('div')`
+  font-size: 1.2em;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/account/accountSubscriptions.jsx]:
```suggestion
  font-size: ${p => p.theme.fontSizeExtraLarge};
```

**Review by markstory — APPROVED** [review]
With Priscila's changes this looks good.

**Pull request merged**